application/src/main/resources/ecc.yml

```diff
@@ -237,6 +237,18 @@ lock_factory:
     ## The keyspace used for the CAS lock factory tables.
     ##
     keyspace: ecchronos
+    ##
+    ## The duration for which the lock is held, in seconds
+    ##
+    lock_time_in_seconds: 600
+    ##
+    ## The interval at which the lock is updated, in seconds
+    ##
+    lock_update_time_in_seconds: 60
+    ##
+    ## The time after which the lock cache expires, in seconds", in seconds
```

Comment:
remove ", also might be worth clarify that this is the failure cache. I.e if we fail to take a lock, all attempts will be failed until it expires.

Reply:
done!